### ISSUE TITLE
Failed to install npm:typings

### ISSUE THREAD

**Issue by kayue** [jspm/jspm-cli#1679]
Hi

Today JSPM failed to install `jspm install npm:typings`, I have tried on both `0.16.31` and `0.16.32` of JSPM.

```
err  Error looking up npm:archy.
```

Sometime I got the following, depends on did I install with cache or not `-f`

```
err  Error looking up npm:chalk.
```

Is there any quick workaround for this?

**Comment by jordie23** [comment]
I'm getting the same, but on `npm:through`. Was getting it on `npm:whatwg-fetch` before.

Installing with npm itself works though? JSPM seems to be having problems.

**Comment by Dillonmcewan** [comment]
I'm getting this as well. It seems to be failing on random packages. Also seeing "Unable to parse package.json" on each of the packages that fails.

**Comment by jordie23** [comment]
> Also seeing "Unable to parse package.json" on each of the packages that fails.

Same. This is the full output I see:

```
warn Error on lookup for npm:through
     Unable to parse package.json

err  Error looking up npm:through.

warn Installation changes not saved.
```

**Comment by calbertts** [comment]
Same issue here, JSPM is failing with a lot of packages that seems to be working fine with NPM.

**Comment by xiankai** [comment]
Having the same problem as well, random packages failing on codeship build process but not on my local install oddly enough.

Was using 0.16.32

**Comment by lookfirst** [comment]
Same here!

**Comment by AdamBurgess** [comment]
If you're on 0.17, `npm install --save https://github.com/AdamBurgess/npm.git` should fix it for now until jspm/npm#133 is accepted
note I think this is not a bug with JSPM but rather a misconfigured NPM server (it's sending compressed content when compression isn't requested)

if you're using 0.16 then you should upgrade ;)

**Comment by calbertts** [comment]
Is it safe to upgrade to 0.17 since it's a beta version?
I've tried but I've got a warning message, it seems the configuration project has changed.

When 0.17 is gonna be released?

**Comment by tom10271** [comment]
```
warn Error on lookup for npm:commander
     Unable to parse package.json

warn Error looking up npm:commander.
     Looking up npm:microplugin
```

Need to disable SystemJS buildCSS property

**Comment by MTRo** [comment]
Hi.

I fixed this issue.

jspm-npm/npm.js (160)
```
      return asp(request)(auth.injectRequestOptions({
        uri: self.registryURL(scope) + '/' + repoPath,
        strictSSL: self.strictSSL,
        headers: lookupCache ? {
          'if-none-match': lookupCache.eTag
        } : {}
      }, doAuth && self.auth)).then(function(res) {
```
To this.
```
      return asp(request)(auth.injectRequestOptions({
        uri: self.registryURL(scope) + '/' + repoPath,
        strictSSL: self.strictSSL,
        gzip : true,
        headers: lookupCache ? {
          'if-none-match': lookupCache.eTag
        } : {}
      }, doAuth && self.auth)).then(function(res) {
```

Add "gzip : true" option for Http request of "request" module options.

**Comment by MTRo** [comment]
I wrote a comment regarding to this issue at below.
It will be fixed soon.  I think..

[https://github.com/jspm/npm/issues/133]

**Comment by guybedford** [comment]
The gzip change has been released - an update to jspm should bring this fix through now.

**Comment by glen-84** [comment]
How do we get the gzip fix into JSPM 0.16.x ?

I don't understand what `an update to jspm should bring this fix through now` means.

**Comment by glen-84** [comment]
Thanks to @AdamBurgess & @guybedford, I just re-installed my current version of JSPM using:

`npm i jspm@0.16.31 -DE`

... and it pulled in the new version of `jspm-npm`.
